### DISCUSSION TITLE
How to generate the athena++ logo mesh

### DISCUSSION THREAD

**Discussion by kurtsansom** [PrincetonUniversity/athena#372] [Q&A]
I would like to replicate the Athena++ logo image, does the input for necessary files exist in the code base to make that image?

**Comment by kurtsansom** [comment]
maybe add a scalar field and populate it in the pattern you want?

**Comment by c-white** [comment]
The input files were never added to the codebase, since they didn't directly pertain to science. I can't seem to find a copy right now, but I can describe the idea.

The pgen read in a black and white BMP file and, for each cell in the 2D polar grid, found the closest pixel in the 2D Cartesian BMP image. Then it enrolled a strong cooling function that only operated on cells where the image was black. With that in place, you can trigger all sorts of hydrodynamics and/or adaptive mesh refinement, while still making the original image somewhat visible in a plot of temperature.

As an aside, for such a simple idea the BMP format has an enormous number of variations, and I'm pretty convinced many graphics programs violate the standards anyway. Portability was always tricky with this pgen.

If anyone has the original files, maybe they can share. Otherwise maybe they're on one of the many computers where I have a copy of Athena.
